app/networkapi/people/models.py

```diff
@@ -97,6 +97,12 @@ class Person(SortableMixin):
         null=True,
         blank=True,
     )
+    interview_url = models.URLField(
+        max_length=500,
```

Comment:
Interestingly from the docs: `Like all CharField subclasses, URLField takes the optional max_length argument. If you don’t specify max_length, a default of 200 is used.`